### PR TITLE
Fix broken templates for RISC-V and RISC-V64

### PR DESCRIPTION
**Pull request by SurajSonawane2415**

### CODE REVIEW

File: templates/riscv64/minimal/mods.conf [hunk 1]
```diff
@@ -15,7 +15,7 @@ configuration conf {
 	@Runlevel(1) include embox.driver.interrupt.riscv_plic
 
 	/* include embox.driver.interrupt.no_clock */
-	include embox.driver.clock.riscv_clk(base_mtime=0x200bff8, base_mtimecmp=0x2004000, rtc_freq=10000000)
+	include embox.driver.clock.riscv_clk(rtc_freq=10000000)
```
Comment (anton-bondarev) [inline templates/riscv64/minimal/mods.conf:18]: Missed` include embox.driver.interrupt.riscv_clint`

File: templates/riscv64/qemu-smode/mods.conf [hunk 2]
```diff
@@ -24,7 +24,7 @@ configuration conf {
 	include embox.driver.diag(impl="embox__driver__serial__ns16550_diag")
 
 	include embox.driver.interrupt.riscv_plic(threshold_offset=0x201000, interrupt_enable_offset=0x2080)
-	include embox.driver.clock.riscv_clk(base_mtime=0x200bff8, base_mtimecmp=0x2004000, rtc_freq=10000000)
+	include embox.driver.clock.riscv_clk(rtc_freq=10000000)
```
Comment (anton-bondarev) [inline templates/riscv64/qemu-smode/mods.conf:27]: Missed `include embox.driver.interrupt.riscv_clint`

**Comment by anton-bondarev** [comment]
@SurajSonawane2415 Hello, Thanks
But I see some other using riscv_clk
```
templates/riscv/sifive_e/mods.conf:19:	include embox.driver.clock.riscv_clk(base_mtime=0x200bff8, base_mtimecmp=0x2004000)
templates/riscv/minimal/mods.conf:18:	include embox.driver.clock.riscv_clk(base_mtime=0x200bff8, base_mtimecmp=0x2004000)
templates/riscv/sifive_u/mods.conf:21:	include embox.driver.clock.riscv_clk(base_mtime=0x200bff8, base_mtimecmp=0x2004000, rtc_freq=10000000)
templates/riscv/qemu/mods.conf:26:	include embox.driver.clock.riscv_clk(base_mtime=0x200bff8, base_mtimecmp=0x2004000, rtc_freq=10000000)
templates/riscv64/maixbit/mods.conf:26:	include embox.driver.clock.riscv_clk(base_mtime=0x200bff8, base_mtimecmp=0x2004000, rtc_freq=10000000)
templates/riscv64/minimal/mods.conf:18:	include embox.driver.clock.riscv_clk(base_mtime=0x200bff8, base_mtimecmp=0x2004000, rtc_freq=10000000)
templates/riscv64/qemu-smode/mods.conf:27:	include embox.driver.clock.riscv_clk(base_mtime=0x200bff8, base_mtimecmp=0x2004000, rtc_freq=10000000)
templates/riscv64/qemu/mods.conf:26:	include embox.driver.clock.riscv_clk(base_mtime=0x200bff8, base_mtimecmp=0x2004000, rtc_freq=10000000)
platform/maix/templates/maix_bit/mods.conf:26:	include embox.driver.clock.riscv_clk(base_mtime=0x200bff8, base_mtimecmp=0x2004000, rtc_freq=10000000)
platform/vostok/templates/vostok_vg015_dev/mods.conf:19:	@Runlevel(1) include embox.driver.clock.riscv_clk(base_mtime=0x0200bff8, base_mtimecmp=0x02004000,rtc_freq=50000000)
```
I think I should also add 'depends embox.driver.interrupt.riscv_clint` into embox.driver.clock.riscv_clk module description

**Comment by SurajSonawane2415** [comment]
@anton-bondarev  Thanks for the review and suggestions. I’ve made all the changes. Could you please review them?

**Review by anton-bondarev — APPROVED** [review]
LGTM

**Pull request merged**